pylint_odoo/checkers/modules_odoo.py

```diff
@@ -360,6 +360,8 @@ def _is_absolute_import(node, name):
                 modnode is not importedmodnode and \
                 importedmodnode.name != name:
             return True
+        if node.as_string().split()[1].startswith('.'):
```

Comment:
Is a absolute import If starts with `.` ?

Reply:
@moylop260 
I change this if statement to another location in favor to keep the `method _is_absolute_import` without change